tests/situations/commun/vues/qcm.test.js

```diff
@@ -49,71 +49,81 @@ describe('La vue de la question QCM', function () {
     });
 
     it("Sait espacer les chiffres dans l'input pour les question de type numérique", function () {
       question.espacerChiffres = false;
       const conteneurChiffresNonEspaces = uneQuestionNumerique().find('.numerique-input-conteneur');
-      expect(conteneurChiffresNonEspaces.classes('chiffres-espaces')).to.be(false);
+      expect(conteneurChiffresNonEspaces.classes('chiffres-espaces')).toBe(false);
     });
   });
 
   it("affiche un bouton d'envoi de réponse", function () {
     const vue = shallowMount(VueQCM, { localVue, propsData: { question } });
-    expect(vue.contains('.question-bouton')).to.be(true);
+    expect(vue.find('.question-bouton').exists()).toBe(true);
   });
 
-  it("désactive le bouton lorsqu'aucune réponse n'est sélectionnée", function () {
+  it("désactive le bouton lorsqu'aucune réponse n'est sélectionnée", function (done) {
     question.choix = [{ id: 'uid-32' }];
     const vue = shallowMount(VueQCM, { localVue, propsData: { question } });
-    expect(vue.find('.question-bouton').attributes('disabled')).to.eql('disabled');
+    expect(vue.find('.question-bouton').attributes('disabled')).toEqual('disabled');
 
     vue.find('input[type=radio][value=uid-32]').setChecked();
-    expect(vue.find('.question-bouton').attributes('disabled')).to.be(undefined);
-  });
+    vue.vm.$nextTick(() => {
+      expect(vue.find('.question-bouton').attributes('disabled')).toBe(undefined);
+      done();
+    });
+  }
+  );
 
   it('rapporte son ouverture au journal', function (done) {
     question.metacompetence = 'ma métacompétence';
     localVue.prototype.$journal.enregistre = (evenement) => {
-      expect(evenement).to.be.a(EvenementAffichageQuestionQCM);
-      expect(evenement.donnees()).to.eql({ question: question.id, metacompetence: 'ma métacompétence' });
+      expect(evenement).toBeInstanceOf(EvenementAffichageQuestionQCM);
+      expect(evenement.donnees()).toEqual({ question: question.id, metacompetence: 'ma métacompétence' });
       done();
     };
     shallowMount(VueQCM, { localVue, propsData: { question } });
   });
 
-  it('emet un événement réponse quand on appuie sur le bouton envoi', function () {
+  it('emet un événement réponse quand on appuie sur le bouton envoi', function (done) {
     question.choix = [{ id: 'uid-32', bonneReponse: true }];
     const vue = shallowMount(VueQCM, { localVue, propsData: { question } });
     vue.find('input[type=radio][value=uid-32]').setChecked();
-    vue.find('.question-bouton').trigger('click');
-    expect(vue.emitted('reponse').length).to.eql(1);
-    expect(vue.emitted('reponse')[0][0]).to.eql({ reponse: 'uid-32', succes: true });
-  });
+    vue.vm.$nextTick(() => {
+      vue.find('.question-bouton').trigger('click');
+      vue.vm.$nextTick(() => {
+        expect(vue.emitted().reponse.length).toEqual(1);
+        expect(vue.emitted().reponse[0][0]).toEqual({ reponse: 'uid-32', succes: true });
+        done();
+      });
+    });
+  }
+  );
```

Comment:
transformer en une ligne : 
```suggestion
  });
```

Reply:
fait